### PR TITLE
Change dash image

### PR DESCRIPTION
**Pull request by AgustinBRodriguez**

### CODE REVIEW

**Comment by AgustinBRodriguez** [comment]
Proposed changes to the General Settings Page on the Dashboard. 
1.settings should be renamed configuration
2.advanced should be renamed firmware.  
ss_en_choices_0

**Comment by AgustinBRodriguez** [comment]
Proposed Navigation bar changes
1. Replace Amiga SDK with Amiga DOCS
2. Replace Introduction with User Manual
3. Replace Concepts with Developers and marge API into it 
4. Examples Stays the same
5. About Us goes away
6. Discuss and Shop stay the same.

**Comment by AgustinBRodriguez** [comment]
reached out to James for missing png's (updator app, auto control blue/yellow outlines, cruise control green outline, e-stop red outline, unlocked pendant etc),  and a brief written description on these line items
wheel_track
v_max_default
max_turn_rate
min_turn_rate
turtle_v
turtle_turn_rate
max_accel
flip_joystick
steering_gamma
batt_lo
batt_hi
pto_gear_ratio
default_pto_rpm
min_pto_rpm
max_pto_rpm
m10_on (motor A on)
m11_on (motor B on)
m12_on (motor C on)
m13_on (motor D on)

Once those images/data arrives a new revision will take place, hopefully with an updated Dashboard overview video.

**Review by guilhermedemouraa — APPROVED** [review]

File: website/docs/dashboard/dashboard_user_guide.md [hunk 1]
```diff
@@ -3,11 +3,12 @@ id: dashboard-user-guide
 title: Dashboard User Guide
 ---
 :::caution
-This is subject for change with new updates to the Dashboard.
-The user guide represents Dashboard Firmware release `v0.1.9`.
-The latest version is Dashboard Firmware release `v0.4.0`.
+This guide is subject to change with each new update to the Dashboard.
+
+The latest version is Dashboard Firmware release `v0.4.2`.
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:8]: 0.4.2 is an internal release, and is not publicly available

```suggestion
The latest version is Dashboard Firmware release `v0.4.0`.
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 2]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
 
-:::tip
-This button will take you to the same page as the `pendant` section of the Settings tab.
-:::
+This is the home screen of the dashboard. In the center sits the spedometer,
+ where you can switch between metric and standard units and adjust to the travel speed of your Amiga.
 
-If your pendant is not being detected at all and if it says `No Pendant Detected`
-please contact the Farm-ng Support immediately by visiting [our Support page](https://amiga.farm-ng.com/docs/support/).
+git  On the right side of the screen you will see several icons displaying:
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:37]:
```suggestion
On the right side of the screen you will see several icons displaying:
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 3]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
-     width="330"
-     height="300" />
+ You will also see details on each motor's:
 
-This screen for monitoring each motor in much more detail than that of the home screen.
-On this screen you can:
+- Voltage
+- RPM
+- Temperature
+- Current draw for each motor/controller
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:62]:
```suggestion
- Voltage
- RPM
- Temperature
- Current draw
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 4]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
-     height="300" />
+- Active
+- Cruise
+- Auto Ready/Active
+- E-Stopped
 
-<img src="https://user-images.githubusercontent.com/64480560/232093025-48b6792e-08a3-48f8-b83e-52d4813d84e9.jpg"
-     width="330"
-     height="300" />
+ You will also see details on each motor's:
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:57]:
```suggestion
 You will also see details on each motor / motor controller's:
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 5]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
-On this screen you can:
+- Voltage
+- RPM
+- Temperature
+- Current draw for each motor/controller
 
-- Monitor the control state of the Amiga
-- Voltage, RPM, temperature, and current of each motor / motor controller
-- State of the CAN bus
+Lastly, CANbus states appear on the right side of the display.
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:64]:
```suggestion
Lastly, CAN bus states appear on the right side of the display.
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 6]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
+Follow this link for an indepth description of your Amiga's
+[Canbus Service](https://amiga.farm-ng.com/docs/concepts/canbus_service/)
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:66]: That link is for the brain interacting with the can bus. It's not really applicable to the CAN states IMO 

```suggestion
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 7]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
-     width="320"
-     height="300" />
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/9a8dcddf-cb5d-4e3c-95e0-0224f521ae6d"/>
 
-This screen is where the autonomous control of the amiga comes to life. Here you will be able to :
+If you have an [Intelligence Kit](/docs/intelligence-kit/overview-intel)
+ mounted on your Amiga and have some tracks recorded in
+[Autoplot](https://amiga.farm-ng.com/docs/apps/autoplot_app/), it is here where autonomous control
+comes to life.
+From this screen, you will be able to:
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:76]:
```suggestion
If you have an [Intelligence Kit](/docs/intelligence-kit/overview-intel)
mounted on your Amiga and have some tracks recorded in
[Autoplot](https://amiga.farm-ng.com/docs/apps/autoplot_app/), 
activating the auto mode on this screen is what brings autonomous control into action.

From this screen you will be able to:
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 8]
```diff
@@ -16,96 +17,90 @@ title="YouTube video player" frameborder="0"
...
-- Monitor the auto control state
-- Monitor the commanded & measured speed and turning rates of the amiga
+- Monitor the Auto Control state
+- Monitor the commanded & measured speed of your Amiga
 
-If you are controlling the Amiga with a brain or other micro-controller,
-here is where you will enable & monitor the auto control commands sent to your Amiga.
-Auto control examples include the [FPV example](/docs/examples/FPV/) (micro-controller).
+You can use other micro-controllers to command your Amiga.
+For an example of how to do so, please follow the instructions in this [FPV example](/docs/examples/FPV/)
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:83]:
```suggestion
You can also use micro-controllers to command your Amiga, like our [Microcontroller Kit](/docs/mcu_kit/).
For an example of how to do so, please follow the instructions in this [Hello Main Loop example](/docs/examples/hello_main_loop/) or this [FPV example](/docs/examples/FPV/)
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 9]
```diff
@@ -128,96 +123,80 @@ The settings you can set on this page are:
...
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bd24caf0-01c7-4647-b6ba-9ce6c55c47aa"/>
 
-<img src="https://user-images.githubusercontent.com/53625197/233201899-59ddb864-75dd-45d3-86c9-bdb7b3c4a358.jpg"
-     width="320" />
+An H bridge circuit allows you to apply a voltage across a load (i.e. a linear actuator)
+in either direction. We currently support up to four h bridge modules on the Amiga.
+They can be engaged momertarily or lached.
+The `up` - `down` arrows are used to `Active` the H bridge
+and the `left` - `right` arrow keys are used to select between H-bridge devices.
+As of this writing only one H-bridge device can be actuated at a time.
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:173]: I thought we can couple in the released fw? If not, we can say starting with `v0.5.0` (unless there is a bug that needs to be fixed?)
```suggestion
Two H-bridge devices (0 & 1 or 2 & 3) can be coupled with the **coupler** button and actuated together.
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 10]
```diff
@@ -128,96 +123,80 @@ The settings you can set on this page are:
...
-> If you see the "No PTO" message and expect a PTO device, contact farm-ng for support by visiting
-> [our Support page](https://amiga.farm-ng.com/docs/support/).
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bd24caf0-01c7-4647-b6ba-9ce6c55c47aa"/>
 
-<img src="https://user-images.githubusercontent.com/53625197/233201899-59ddb864-75dd-45d3-86c9-bdb7b3c4a358.jpg"
-     width="320" />
+An H bridge circuit allows you to apply a voltage across a load (i.e. a linear actuator)
+in either direction. We currently support up to four h bridge modules on the Amiga.
+They can be engaged momertarily or lached.
+The `up` - `down` arrows are used to `Active` the H bridge
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:171]:
```suggestion
They can be engaged momentarily or latched.
The `up` - `down` arrows are used to control the selected H bridge device(s)
```

File: website/docs/dashboard/dashboard_user_guide.md [hunk 11]
```diff
@@ -229,6 +208,64 @@ For more information about Firmware Updates or how to perform an update,
...
+
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/e08dd594-e229-4820-904c-9e4f9aeb2bc1"/>
+
+Occasionally a message stating that no PTO device has been detected will be accompanied by the
+following image.
+Inspect all cables and make sure they are properly connected.
+<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bf3d2a67-d10b-41ca-b35b-89fc0f3a1199"/>
+
+If you expect a PTO device but it is not showing up,
+ please reach out to us via farm-ng's
```
Comment (Hackerman342) [inline website/docs/dashboard/dashboard_user_guide.md:240]:
```suggestion
please reach out to us via farm-ng's
```

**Review by Hackerman342 — APPROVED** [review]
Great changes! Just some nitpicks to address

**Pull request merged**